src/sgx/faasm_sgx_attestation.cpp

```diff
@@ -21,6 +28,12 @@
 extern "C"
 {
     extern __thread faaslet_sgx_msg_buffer_t* faasletSgxMsgBufferPtr;
+    extern sgx_status_t sgx_ra_get_ga(unsigned long, _status_t *, unsigned int, _sgx_ec256_public_t *);
+    extern sgx_status_t faasm_sgx_enclave_init_ra(sgx_enclave_id_t, faasm_sgx_status_t*, sgx_ra_context_t*);
+    extern sgx_status_t sgx_ra_proc_msg2_trusted(unsigned long, _status_t *, unsigned int, const _ra_msg2_t *, const _target_info_t *, _report_t *, _quote_nonce *);
+    extern sgx_status_t sgx_ra_get_msg3_trusted(sgx_enclave_id_t eid, sgx_status_t* retval, sgx_ra_context_t context, uint32_t quote_size, sgx_report_t* qe_report, sgx_ra_msg3_t *p_msg3, uint32_t msg3_size);
+    extern sgx_status_t faasm_sgx_enclave_finalize_key_exchange(sgx_enclave_id_t, faasm_sgx_status_t* , sgx_wamr_msg_t*, uint32_t);
```

Comment:
We use 80 char line limits in this project, this should be tidied by `run_clang_format.sh`